content-repo/extra-docs/articles/Default Pack.md

```diff
@@ -0,0 +1,88 @@
+---
+title: The Default Pack
+description: Overview of the Default pack in Cortex XSOAR.
+---
+
+## Introduction
+
+The Default pack provides a solution for new users who are still early in their Cortex XSOAR journey, or as a solution for use cases that you would like to handle but are yet to create content for.
+
+  
+
+We’ve created a playbook that puts together the core elements of handling any kind of incident. With this playbook there’s also a new layout that gives insight into the data that came with the incident together with any new information collected during the investigation of that incident.
+
+  
+  
+
+## Walkthrough
+
+### Playbook
+
+The playbook makes use of existing out-of-the-box, free content in order to investigate the incident.
+
+It has 3 inputs - whether to extract and enrich indicators from the fetched incident (turn on in case you decided to disable [auto-extract](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.6/Cortex-XSOAR-Administrator-Guide/Indicator-Extraction)), and two tags used for allowing or blocking indicators using an External Dynamic List.
+
+Extracting and enriching the indicators related to the incident is crucial for the investigation and response process. If you are unsure, we suggest to keep the input value as "True".
+
+The tag inputs are optional names that you would like to use for tagging indicators. For example, you may want to tag malicious indicators as "mal", or benign indicators as "safe". The Default layout then allows you to make use of these tags by using the buttons that correspond to the indicators you want to tag.
+Note: Tagging the indicators does not automatically allow or block them. Instead, tagging indicators can be used in conjunction with an External Dynamic List and the Generic Export Indicators Service integration. You can read more about it by [installing the pack](https://cortex.marketplace.pan.dev/marketplace/details/EDL/) from Marketplace and reading the integration documentation..
+
+The playbook begins by de-duplicating any similar incidents. If a file is involved in the incident, it will then run the following steps:
```

Comment:
I think you should mention that only 1-4 are relevant if a file is involved. Otherwise it sounds like all are related to the file.

Reply:
True, fixing